of.go

```diff
@@ -0,0 +1,1895 @@
+package future
```

Comment:
Using go generate and add generated file header?

Reply:
fix